tofu/scripts/tofucalc.py

```diff
@@ -13,8 +13,9 @@
 # tofu
 # test if in a tofu git repo
 _HERE = os.path.abspath(os.path.dirname(__file__))
+_HERE = os.path.dirname(os.path.dirname(_HERE))
```

Comment:
where is _HERE first defined ?

Reply:
one line above

Reply:
Here we are taking _HERE as the directory of the module:
`_HERE = os.path.abspath(os.path.dirname(__file__))`

and then we go 2 parents directories up:
`_HERE = os.path.dirname(os.path.dirname(_HERE))`

That's because tofuplot and tofucalc were moved from the root directory down to tofu/scripts

Reply:
yes sorry, somehow i didn't see the line above (twice)